### PR TITLE
Add cron job to test ApacheOnly builds

### PR DESCRIPTION
**Pull request by JLockerman**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -153,6 +153,22 @@ jobs:
...
+      env: PG_VERSION=11.0 PG_GIT_TAG=REL_11_0 OTHER_CMAKE_FLAGS="-DAPACHE_ONLY=true"
+
+    - if: (type = cron) OR (branch = prerelease_test)
+      stage: test
+      name: "ApacheOnly regression tests 10.2"
+      env: PG_VERSION=10.2 PG_GIT_TAG=REL_10_2 OTHER_CMAKE_FLAGS="-DAPACHE_ONLY=true"
+
+    - if: (type = cron) OR (branch = prerelease_test)
+      stage: test
+      name: "ApacheOnly regression tests 10.2"
```
Comment (RobAtticus) [inline .travis.yml]: `10.2` -> `9.6.6`

**Comment by JLockerman** [comment]
tests pass https://travis-ci.org/timescale/timescaledb/builds/476508379

**Comment by JLockerman** [comment]
@RobAtticus fixed nit, but leaving it as 3 commits since they are logically separable.

**Comment by RobAtticus** [comment]
Does each commit build on its own? If not I would combine them until they do. Ideally we should be able to checkout any commit in master and have it build.

**Comment by JLockerman** [comment]
@RobAtticus yup

**Comment by JLockerman** [comment]
first change fixes the build, second change makes the build system more backwards compatible with older cmake, the third adds the cron job

**Pull request merged**